src/molecules/topNavbar/TopNavbar.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import DeliveryHead from "../../atoms/deliveryHead";
```

Comment:
We can define all these atoms inside topNavbar folder since only one component is using it